sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs

```diff
@@ -549,10 +432,10 @@ public partial class CustomRule
         public static bool operator !=(Azure.ResourceManager.FrontDoor.Models.DynamicCompressionEnabled left, Azure.ResourceManager.FrontDoor.Models.DynamicCompressionEnabled right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class Endpoint
+    public partial class EndpointProperties
```

Comment:
Should this has a prefix? What kind of endpoint properties is this thing applying to?

Reply:
Change the name to `FrontDoorExperimentEndpointProperties`